orte/mca/schizo/base/schizo_base_stubs.c

```diff
@@ -46,6 +46,11 @@ int orte_schizo_base_parse_env(char *personality,
     int rc;
     orte_schizo_base_active_module_t *mod;
 
+    if (NULL == personality) {
+        opal_output(0, "NULL PERSONALITY");
```

Comment:
I can make it an error log - I just wanted something to tell us that a condition which should never occur happened so we can fix it.